app/controllers/application_controller.rb

```diff
@@ -8,7 +8,7 @@ class ApplicationController < ActionController::Base
   rescue_from ActiveRecord::RecordNotFound, with: :not_found
 
   before_action :redirect_to_canonical_domain, :set_headers
-  before_action :store_jobseeker_redirect_to!, if: -> { redirect_to_param.present? }
```

Comment:
Why deleted?

Reply:
My bad I thought we'd stopped using it, but we haven't!!